opengrok-web/src/main/java/org/opengrok/web/api/v1/controller/ConfigurationController.java

```diff
@@ -68,6 +68,7 @@ public String get() {
 
     @PUT
     @Consumes(MediaType.APPLICATION_XML)
+    @Produces("text/plain")
```

Comment:
Why these? Does it actually matter when there is no output?

Reply:
At one point Grizzly complained about not having the media type however this does not seem to be the case anymore. Reverting.